tests/baselines/reference/computedPropertyNamesContextualType8_ES5.errors.txt

```diff
@@ -1,20 +1,21 @@
-tests/cases/conformance/es6/computedProperties/computedPropertyNamesContextualType8_ES5.ts(7,5): error TS2418: Type of computed property's value is 'string', which is not assignable to type 'boolean'.
-tests/cases/conformance/es6/computedProperties/computedPropertyNamesContextualType8_ES5.ts(8,5): error TS2418: Type of computed property's value is 'number', which is not assignable to type 'boolean'.
+tests/cases/conformance/es6/computedProperties/computedPropertyNamesContextualType8_ES5.ts(6,5): error TS2322: Type '{ [x: string]: string | number; }' is not assignable to type 'I'.
+  Index signatures are incompatible.
+    Type 'string | number' is not assignable to type 'boolean'.
+      Type 'string' is not assignable to type 'boolean'.
 
 
-==== tests/cases/conformance/es6/computedProperties/computedPropertyNamesContextualType8_ES5.ts (2 errors) ====
+==== tests/cases/conformance/es6/computedProperties/computedPropertyNamesContextualType8_ES5.ts (1 errors) ====
     interface I {
         [s: string]: boolean;
         [s: number]: boolean;
     }
     
     var o: I = {
+        ~
+!!! error TS2322: Type '{ [x: string]: string | number; }' is not assignable to type 'I'.

```

Comment:
Any idea what's going on here? I'm not sure why we stopped issuing the per-property error.

Reply:
I assumed it was the change in looking up the type of property names.

Reply:
It's because we previously treated _all_ computed properties as having known (unit type) names, when here they actually have type `string`. Since they now have type `string` we generate a string index signature, and the elaboration logic stops trying to drill down to the properties.